en/python_introduction/README.md

```diff
@@ -252,7 +252,11 @@ This shows that you just created an empty dictionary. Hurray!
 
 Now, try writing the following command (try replacing your own information too):
 
-    >>> participant = {'name': 'Ola', 'country': 'Poland', 'favorite_numbers': [7, 42, 92]}
```

Comment:
I'm inclined to think whitespace is better introduced at the `if` stage when we've started saving to files rather than doing it on the prompt - it seems like an extra complication at this point in the tutorial. I can see people getting unnecessarily frustrated, or trying to type in the dots.